add reference to GUI library import (#31)
lennessyy/graph-ml: modules/using-an-algorithm/pages/index.adoc

The algorithms are divided into several categories.
Each algorithm has its own folder, where you can find the implementation of the algorithm, any user-defined functions required to install the algorithm, or different versions of the algorithm if they exist.

Some query algorithms rely on subqueries, which need to be created as separate queries and must be installed together.

GDS Library algorithms are implemented in GSQL as GSQL queries.
Therefore, the process to run an algorithm query is the same as the process to run any other query.

The algorithm library is included, ready to install, with the TigerGraph database.

> Database versions 3.8 and newer offer the library in an additional format, as packaged template queries.

Reviewer: ```suggestion
Database versions 3.8 and higher offer the library in an additional format, as packaged template queries.
```

== Packaged template queries

If your version of TigerGraph Server is 3.8 or higher, you can use packaged queries to run most Graph Data Science Library algorithms.

> Packaged queries cannot be run from GraphStudio.

Reviewer: I would move this sentence after the following two sentences. That is, first explain how a user CAN run packaged queries. Then mention that they can't use GraphStudio.

----

xref:gui:graphstudio:overview.adoc[GraphStudio] version 3.8 does not support using the `CALL` command for packaged template queries.

> To install queries from the library on GraphStudio, you can either import them from the GUI or copy and paste the query source code from the TigerGraph GitHub repository.

Reviewer: "import them from the GUI" doesn't make sense to me.  I think you are talking about the method that uses the Add Query/New GSQL Query button (+ icon) and then the "Choose From Library" button.  That is a bit of a mouthful, we need decide what is our official/standard name for that method and use it consistently. Use it in a way that people realize you are **naming** a specific method, and not just using English to **describe** something.

Reviewer: Overall, this page needs to address two types of algorithms. For non-packaged queries, there are two methods:

1. If your version of TG supports Packaged Queries, then read this for <how to install and use packaged queries>.
Not all library algorithms are currently available as packaged queries.
2. You can always use non-packaged queries. 
In GraphStudio, you can chose to install algorithms via the Add Query. Then either
(a) Click the Choose From Library button and just click the algorithms you want. Note: not all algorithms are available here.
(b) Copy and paste the GSQL code for the algorithm from GitHub into a GSQL query editing pane. Note: some algorithms have subqueries which need to be created as separate queries and installed together.

. Run `@<file path>` from the GSQL shell, and replace `<file path>` with the absolute path to the file where you copied the query.
For example, if your filepath is `/home/tigergraph/query/pagerank.gsql`, run `@/home/tigergraph/query/pagerank.gsql` from the GSQL shell.
--
GraphStudio Library Import::
+
--
This method is easier than copying and pasting the source code, but not all queries, subqueries, and query variants are available using this method.

> Follow the instructions in xref:gui:graphstudio:write-queries.adoc[] to open the Write Query window and select a query from the library through the GUI.

Reviewer: "Write Query window" doesn't make sense to me.
Did you mean the Write Queries page, or the Add Query command? I think we want to say the Add Query button (with the + sign icon), followed by the "Choose From Library" button.

--
GraphStudio Source Code::
+
--

> This method is more complex than the library import, but gives you access to all queries, subqueries, and query variants that may not yet be available in the library import.

Reviewer: ```suggestion
This method is more complex than the library import, but is suitable for all queries, subqueries, and query variants, including any library algorithms that may not yet be available via the library import method.
```
